src/typescript.ts

```diff
@@ -1,33 +1,41 @@
 import type { CompilerOptions } from 'typescript'
 import { resolve, dirname } from 'path'
+import { promises as fsp } from 'fs'
 import { Module } from 'module'
-import { exit, fileExists } from './utils'
+import pc from 'picocolors'
+import { exit, fileExists, memoize } from './utils'
+import { DEFAULT_TS_CONFIG } from './constants'
+import { logger } from './logger'
 
 export type TypescriptOptions = {
-  tsConfigPath: string | undefined
+  tsConfigPath?: string
```

Comment:
It's a required instead of optional prop
```suggestion
  tsConfigPath: string | undefined
```